.github/workflows/unit-tests.yml

```diff
@@ -76,6 +76,11 @@ jobs:
           &&
           sudo apt install -y libsqlite3-dev libcurl4-openssl-dev valgrind lcov clang libsystemd-dev
 
+      - name: Install GStreamer
+        run: |
+           sudo apt update
+           sudo apt install -y libunwind-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev
```

Comment:
Do we need the gstreamer packages now?

Reply:
I realized later that this is required to satisfy the gstreamer api uses in the Speaker implementation